src/components/ChatEntry.js

```diff
@@ -1,22 +1,48 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const [isLiked, setIsLiked] = useState(props['liked']);
+
+  // console.log(props)
+
+  const likeButtonClick = () => {
+    setIsLiked( !isLiked );
+    const messageData= {
+      id: props['id'],
+      body: props['body'],
+      sender: props['sender'],
+      timeStamp: props['timeStamp'],
+      liked: !isLiked,
+    };
+    props.updatedMessage(messageData)
+    // console.log(messageData)
+    
+  }
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className="chat-entry local" key={props['id']}>
```

Comment:
Because we set `key` on the ChatEntry components on line 18 of ChatLog where they're created, we should be able to remove the `key` set here.